### DISCUSSION TITLE
Add support for Angular signal based inputs / outputs / models

### DISCUSSION THREAD

**Discussion by iabu94** [storybookjs/storybook#26906] [Ideas]
### Is your feature request related to a problem? Please describe.

I have created the solution based on the issue. You can see the label input in button component created as signal input

https://stackblitz.com/edit/ng-storybook?file=src%2Fstories%2Fbutton.stories.ts

### Describe the solution you'd like

support all the latest features of Angular v17

### Are you able to assist to bring the feature to reality?

no

**Comment by Marklb** [comment]
Is this a duplicate of #26859, with a repro? My answer there shows that it is already supported. https://github.com/storybookjs/storybook/discussions/26859#discussioncomment-9156834

It looks like support was added in `8.1.0-alpha.4`, so you can upgrade to the alpha or wait on `8.1.0`. You could also manually overwrite the types, because it is only a types issue, but I think upgrading would be the better option to avoid going back and cleaning up the overwritten types later.
